docs/how-to/h-external-k8s-connection.md

```diff
@@ -0,0 +1,72 @@
+# External K8s connection
+
+To connect make the Charmed Kafka K8s brokers reachable from outside the Kubernetes cluster, the charm application will can manage several NodePort services:
+
+```console
+$ kubectl get services -n model
+
+| NAME                             | TYPE      | CLUSTER-IP     | CLIENT-IP   | PORT(S)                                                                         |
+|----------------------------------|-----------|----------------|-------------|---------------------------------------------------------------------------------|
+| kafka-k8s                        | ClusterIP | 10.152.183.77  |             | 65535/TCP                                                                       |
+| kafka-k8s-endpoints              | ClusterIP | None           |             |                                                                                 |
+| kafka-k8s-bootstrap              | NodePort  | 10.152.183.193 |             | 29092:31982/TCP,29093:31262/TCP,29094:30845/TCP,29095:30153/TCP,29096:32601/TCP |
+| kafka-k8s-0-sasl-plaintext-scram | NodePort  | 10.152.183.78  |             | 29092:31211/TCP                                                                 |
+| kafka-k8s-0-sasl-ssl-scram       | NodePort  | 10.152.183.104 |             | 29093:32306/TCP                                                                 |
+| kafka-k8s-1-sasl-plaintext-scram | NodePort  | 10.152.183.125 |             | 29092:31421/TCP                                                                 |
+| kafka-k8s-1-sasl-ssl-scram       | NodePort  | 10.152.183.162 |             | 29093:31710/TCP                                                                 |
+| kafka-k8s-2-sasl-plaintext-scram | NodePort  | 10.152.183.223 |             | 29092:31449/TCP                                                                 |
+| kafka-k8s-2-sasl-ssl-scram       | NodePort  | 10.152.183.117 |             | 29093:30641/TCP                                                                 |
+```
+
+- The `kafka-k8s-bootstrap` NodePort service exposes a selection of ports, with each one being associated with the supported security protocol + authentication mechanism
+    - Supported security protocols include `SASL_PLAINTEXT`, `SASL_SSL` and `SSL`
+    - Supported authentication mechanisms include `SCRAM-SHA-512`, `OAUTHBEARER` and `SSL`
```

Comment:
**todo** I find the bullet points a bit confusing, since I would naively expect to have 3 (security_protocol)*3(auth) ports, instead of 5. I would suggest to just refer to the documentation, like saying:

> The `kafka-k8s-bootstrap` NodePort service exposes a selection of ports, with each one being associated with the supported security protocol + authentication mechanism. Please refer to the listener table for more information to which port refers to which protocol and authentication.

Reply:
Good suggestion, I'll remove the bulletpoints.